templates/Program/programComments.html.twig

```diff
@@ -51,11 +51,23 @@
             <div class="date">
               {{ comment.uploadDate.format('Y-m-d') }}
             </div>
-            <a id="comment-translate-button-{{ comment.id }}" class="comment-translate-button">
+            <span id="comment-translation-button-{{ comment.id }}" class="comment-translation-button catro-icon-button"
+                data-toggle="tooltip" title="Show Translation">
```

Comment:
does the title "show translation show up when you hover your mouse? if this is a user visible string, it has to be in the strings file. ie, `translation.en.yml`

Reply:
Added show and hide translations to translation.en.yml